lib/capybara/rack_test/driver.rb

```diff
@@ -50,6 +50,10 @@ def refresh
     browser.refresh
   end
 
+  def send_keys(*)
+    raise Capybara::NotSupportedByDriverError, "RackTest driver doesn't have the concept of an active element, please call send_keys on a specific element"
```

Comment:
@twalpole I've updated this error with the message you've suggested.

Reading it back: is it true that Rack Test supports key handlers?

Reply:
Good catch -- I hadn't thought that through -- no it doesn't, and calling `send_keys` on an element would just raise a NotImplementedError, so this might as well be the same via the Capybara::Driver::Base default -- sorry about that.